src/content/advanced/configure-dex-in-your-cluster/index.md

```diff
@@ -97,6 +97,9 @@ In this guide, we will use a single app deployment for each cluster that you wan
 
 We'll use the [app platform](https://docs.giantswarm.io/app-platform/) to deploy the app, as it allows us to deploy apps across workload clusters using a single API endpoint. In this example, we create an `App` custom resource (CR) with the parameters to install our [`dex-app`](https://github.com/giantswarm/dex-app) in the desired cluster, and a `ConfigMap` with the configuration values.
 
+The `connectorConfig` format can look different depending on the oidc provider you want to use. Some examples can be found below. 
+Detailed upstream documentation on all currently available connectors and their respective configuration is available [here](https://dexidp.io/docs/connectors/).
```

Comment:
"here" is not an ideal link text. Better use something that will give the reader a clue where they will end up.

```suggestion
Details on all connectors and their respective configuration is available in the [Dex documentation](https://dexidp.io/docs/connectors/).
```